### PR TITLE
Updated Datatables dependency

### PR DESCRIPTION
**Pull request by clintonb**
This fixes a security vulnerability and fixes build failures resulting from dataTables.boostrap.js referencing "datatables.net" instead of "datatables".

ECOM-2314

### CODE REVIEW

**Comment by clintonb** [comment]
@rlucioni @jimabramson 

FYI @feanil @maxrothman

File: bower.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "requirejs": "~2.1.15",
     "underscore": "~1.8.2",
     "bootstrapaccessibilityplugin": "~1.0.4",
-    "datatables": "~1.10.7",
+    "datatables": "~1.10.10",
```
Comment (rlucioni) [inline bower.json]: Any chance we can pin this to 1.10.10 so the version of this package doesn't change under us again?

Comment (clintonb) [inline bower.json]: Updated.

**Comment by rlucioni** [comment]
:+1:

**Comment by jimabramson** [comment]
:+1:  thx for fixing this @clintonb

**Pull request merged**